platform/config/config-reader-yaml/src/test/resources/ldap.yml

```diff
@@ -0,0 +1,63 @@
+--- 
+!ldap
```

Comment:
Did we not want to group together some of these properties as Tracy had them on the confluence page?